### PR TITLE
Profit in card and table

### PR DESCRIPTION
**Pull request by pavelzubov**

### CODE REVIEW

File: src/components/profitability/profitability.js [hunk 1]
```diff
@@ -4,42 +4,86 @@ import classNames from "classnames";
...
+
 const Profitability = ({
   className,
   children,
   value,
   isPositive,
   isNegative,
+  prefix,
+  form,
   ...rest
```
Comment (nkrivous) [inline src/components/profitability/profitability.js]: What does rest mean?

File: src/components/profitability/profitability.js [hunk 2]
```diff
@@ -4,42 +4,86 @@ import classNames from "classnames";
 import PropTypes from "prop-types";
 import React from "react";
 
+const PREFIXES = {
+  ARROWS: {
+    NEGATIVE: String.fromCharCode(8595),
+    POSITIVE: String.fromCharCode(8593)
+  },
+  SCIENCE: {
```
Comment (nkrivous) [inline src/components/profitability/profitability.js]: science -> sign

File: src/components/profitability/profitability.js [hunk 3]
```diff
@@ -4,42 +4,86 @@ import classNames from "classnames";
 import PropTypes from "prop-types";
 import React from "react";
 
+const PREFIXES = {
+  ARROWS: {
```
Comment (nkrivous) [inline src/components/profitability/profitability.js]: arrows->arrow

File: src/components/profitability/profitability.js [hunk 4]
```diff
@@ -4,42 +4,86 @@ import classNames from "classnames";
...
+      ((isPositiveLocal && `${PREFIXES[prefix].POSITIVE} `) ||
+        (isNegativeLocal && `${PREFIXES[prefix].NEGATIVE} `))
+    );
+  };
+
+  switch (form) {
+    case "ellipse":
+      return (
+        <div
+          className={classNames("ellipse", className, {
```
Comment (nkrivous) [inline src/components/profitability/profitability.js]: variant = ["text", "chips"]. default = "text"

File: src/components/profitability/profitability.js (hunk 1 above)
Comment (pavelzubov) [reply src/components/profitability/profitability.js]: I do not know. it was already there.

**Pull request merged**